app/controllers/styleguide_controller.rb

```diff
@@ -20,6 +20,10 @@ def components
     @components = COMPONENTS
   end
 
+  def page_specific
```

Comment:
Hey @johnplayer1982 what does `page_specific` mean? Does it mean it's going to be used only for the `search_results` page?

Reply:
Page specific components, so things that arent used across the whole website, and are specific to a certain area

Reply:
Thanks @johnplayer1982